Pinta.Core/Actions/WindowActions.cs

```diff
@@ -97,19 +95,18 @@ public void AddDocument (Document doc)
 		AddDocumentMenuItem (workspace.OpenDocuments.IndexOf (doc));
 	}
 
-	public void RemoveDocument (Document doc)
```

Comment:
This is kind of a confusing method now since it doesn't indicate which document was removed - maybe it'd be better to just have this class register a handler for `workspace.DocumentClosed` rather than having `ActionHandlers` call this method?